ui/src/VoxettaApp.js

```diff
@@ -60,28 +102,102 @@ export class VoxettaApp extends LitElement {
      */
     displayComponents() {
         switch (this.state) {
-            case States.RECORD_PAGE:
+            case States.ACTIVE_RECORD_PAGE:
+                return html`
+                    ${this.getActiveRecordTemplate()}
```

Comment:
personal nit: this.renderActiveRecordTemplate ?

Reply:
changed for all getTemplate functions